metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/metainfo.xml

```diff
@@ -187,6 +187,14 @@
                   <timeout>600</timeout>
                 </commandScript>
               </customCommand>
+              <customCommand>
+                <name>ZEPPELIN_DASHBOARD_INSTALL</name>
```

Comment:
Should we call this "Zeppelin Notebook Install"?  I **think** that's the preferred Zeppelin nomenclature.